### ISSUE TITLE
Describing Experience

### ISSUE THREAD

**Issue by Viranchee** [Viranchee/Resume#14]
#### iOS App Developer, [Heady LLC][heady], April 2019 - Present | Mumbai

- Principal Architect on **Popviewers** app, in Beta testing
- Showed the clients a demo every week during development
- Clients renewed contract. Strongest indicator of performance

#### Embedded Developer, Intern, [Earth Energy EV][earthEnergy], December 2017 - June 2018 (6 months) | Mumbai

- Developed Bike Electronics
- Attracted investors by building robust system

[heady]: www.fast.com
[indiawyn]: www.fast.com
[earthEnergy]: www.fast.com

**Comment by Viranchee** [comment]
I think from current experience, I should better describe what beneficial stuff got achieved instead of focusing on micro stuff I did. A macro view.

**Comment by Viranchee** [comment]
Eg. In Popviewers, I lead the demo of the app every week to clients. Initial contract was of 3 months and they have sticked with Heady. They believe in my ability to program iOS Apps and renewed the contract.

**Comment by Viranchee** [comment]
Should I really write **Principal Architect**?
I was the only developer on this app. I had the power to make architecture decisions.
I made good architecture decisions. It hasn't failed me.
I have solved bugs pretty quickly. 
Architecture has never come in my way, during development or bug fixing

**Comment by Viranchee** [comment]
Eg. At Earth Energy, I developed the electronics so well, that Rushi (Manager) happily went into Investment meetings with confidence. Helped bring investors in.

**Comment by Viranchee** [comment]
Eg. At Indiawyn Gaming, I demonstrated potential. Manager happily purchased training resources for me.

**Comment by Viranchee** [comment]
About Me:

I'm Viranchee, I build iPhone Apps which users love.

Popviewers, the multimedia app I made, is in Beta, and has favourable reviews.

The clients are happy.

We faced a problem displaying multiple Videos on screen. It would lag up the UI. I simplified and optimized the code that now scrolling is smooth even on lowest end iPhone.

I programmed the UI to work on every iPhone size and hence never had size complaints, and hence, a feature in disguise

I implemented Xcode optimization technique in 15 minutes across the project and saved 5 minutes everyday since the last 6 months. Less time waiting for code to compile = More time to implement new features

Saved developer fatigue of total 75 minutes every day by reducing stand up time from 30 minutes to 5, by having more written and less verbal communication

By utilizing meta programming, I have made time by limiting time doing manual stuff and eliminate common points of failure

By following the usage of agreed upon Style guide,  arguments on code structure reduced significantly.
This lets one see real issues with better judgement
